src/js/components/icons.json

```
@@ -0,0 +1,1298 @@
+{
+  "icons": [
```

Since we only have one array with all icons, we could have only the array on the json file:

```suggestion
[
    "fab fa-500px",
    "fab fa-accessible-icon",
```